grafana-operator/grafana-operator
# Implement feature_toggles from Grafana 8.0 (backport to v3)

## Description
Grafana 8.0 implements a new configuration option called ["feature_toggles"](https://grafana.com/docs/grafana/latest/administration/configuration/#feature_toggles), backported to the v3 branch.

## Type of change

## Checklist


## Verification steps

1. Set config
```yml
  feature_toggles:
    enable: ngalert
```
2. Check that the new alerting is working
